src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (num) {
+      this.result += num;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - When creating a method in an object, you don't need to use the function keyword. Use the method shorthand instead.

BAD EXAMPLE:
add: function(num) {
  // your code
},

GOOD EXAMPLE:
add(num) {
  // your code
},